lib/Jobs/jobExecutionPropTypes.js

```diff
@@ -2,12 +2,12 @@ import PropTypes from 'prop-types';
 
 const jobExecutionPropTypes = PropTypes.shape({
```

Comment:
I'm not sure that we should have the ability to hide search. If the search is not needed in some cases, it seems, it might be more useful to split search and sort functionality and just to combine them in the SearchAndSort component. 
Or if we are going to add the search in the future (we can clarify this requirement), we can disable (not hide) it for all cases because the search is not supported for now.

What do you think about it?